backend/apps/ml_triage/services/models.py

```
@@ -0,0 +1,225 @@
+            'low': 20,
+        }
+        
+        # Base score from predicted priority
+        priority, confidence = self.predict_priority(issue)
+        base_score = priority_weights.get(priority, 50) * confidence
+        
+        # Add hotness factor
+        hotness = issue.calculate_hotness() or 0
+        
+        # Add comment activity factor
+        comment_count = Comment.objects.filter(issue=issue).count()
```

Ruff (0.15.21)

[error] Undefined name `Comment`

(F821)